lib/src/firestore.dart

```diff
@@ -273,9 +277,10 @@ class DocumentSnapshot {
   /// Returns the ID of the snapshot's document
   String get documentID => nativeInstance.id;
 
-  DateTime get createTime => nativeInstance.createTime != null
-      ? DateTime.parse(nativeInstance.createTime)
-      : null;
+  DateTime get createTime =>
+      nativeInstance.createTime != null
+          ? DateTime.parse(nativeInstance.createTime)
+          : null;
```

Comment:
Did `dartfmt` change this?

Reply:
I guess so. It seems other files might be affected by a dartfmt. I personnally always use it before commit

Reply:
I ran dartfmt on it again, it changes many things...I guess I might have used Intellij formatter at some point (which is somehow slightly different...)